Resources/Prototypes/Research/industrial.yml

```diff
@@ -1,5 +1,18 @@
 ﻿# Tier 1
 
+- type: technology
+  id: SalvageFultons
+  name: research-technology-fultons
+  icon:
+    sprite: Objects/Tools/fulton_balloon.rsi
+    state: fulton_balloon
+  discipline: Industrial
+  tier: 1
+  cost: 2500  # Cheap because salvage should really get it.
+  recipeUnlocks:
+    - Fulton
+    - FultonBeacon
```

Comment:
could probably just make this roundstart in the salvagevend if they need it so bad. If putting fultons in the autolathe is an issue could either make a salvage fab or just make them orderable.

Reply:
sure